torch/onnx/utils.py

```diff
@@ -1774,32 +1781,39 @@ def _run_symbolic_function(
         if symbolic_function_group is not None:
             symbolic_fn = symbolic_function_group.get(opset_version)
             if symbolic_fn is not None:
-                attrs = {k: symbolic_helper._node_get(n, k) for k in n.attributeNames()}
+                attrs = {
+                    k: symbolic_helper._node_get(node, k) for k in node.attributeNames()
+                }
                 if _need_symbolic_context(symbolic_fn):
                     # TODO(justinchuby): Refactor how we check for the need of the symbolic context
-                    ctx = _exporter_states.SymbolicContext(_params_dict, env, n, block)
-                    return symbolic_fn(ctx, g, *inputs, **attrs)
+                    ctx = _exporter_states.SymbolicContext(
+                        _params_dict, env, node, block, opset_version
```

Comment:
I know it's not from this PR, but `_params_dict` seems a dangerous global variable here. I wonder how we can make it safer. I also see your previous PR tag it as TODO: remove its usage from other cpp file.

Reply:
It's ancient code. We should look to remove it from future PRs.